pkg/network/setup/netconf.go

```diff
@@ -53,7 +78,9 @@ func (c *NetConf) Setup(vmi *v1.VirtualMachineInstance, launcherPid int, doNetNS
 	}
 
 	netConfigurator := NewVMNetworkConfigurator(vmi, c.ifaceCacheFactory)
-	err := doNetNS(func() error {
+
+	ns := c.nsFactory(launcherPid)
+	err := ns.Do(func() error {
```

Comment:
you can inline these 2, can't you ? Why make a variable out of it ? I'd say everyone assumes a `nsFactory` gets you `ns` s - whatever those are.

Is it being re-used somewhere and I just missed it ?

Reply:
I personally dislike the train chaining of calls, so I try to avoid it.
If you are worry about the efficiency, I am pretty sure the compiler will inline it anyway.